_posts/2014-10-02-sinatra-app-tutorial.markdown

```diff
@@ -39,9 +39,9 @@ You can actually call your Ruby file whatever you'd like. `vote.rb` for instance
 ### Run your app
 
 Go to the directory where you put your app and run `ruby suffragist.rb`.
-Now you can visit [localhost:4567](http://localhost:4567). You should
+Now you can visit <a href="localhost:4567" target="_blank">localhost:4567</a>. You should
 see a ‘Hello, voter!’ page, which means that the generation of your new
-app worked correctly. Hit `ctrl-c` in the terminal to quit the server.
+app worked correctly. Hit `CTRL + C` in the terminal to shut down the server. If `CTRL + C` does not work for you it means you are probably Windows user and `CTRL + Z`/ `Ctrl + Pause` / `Ctrl + Pause` will fix the issue)
```

Comment:
I often see lower cased `ctrl` with uppercase `L`etter on stack exchange websites. 

But as it is your tutorial you’re the boss!

I’ll do it `ctrl+c`
-- 
Pozdrawiam / Regards
Filip Bartuzi

> On 18 Sep 2015, at 13:58, Piotr Szotkowski <notifications@github.com> wrote:
> 
> In _posts/2014-10-02-sinatra-app-tutorial.markdown <https://github.com/railsgirls/railsgirls.github.com/pull/255#discussion_r39851376>:
> 
> >  see a ‘Hello, voter!’ page, which means that the generation of your new
> > -app worked correctly. Hit `ctrl-c` in the terminal to quit the server.
> > +app worked correctly. Hit `CTRL + C` in the terminal to shut down the server. If `CTRL + C` does not work for you it means you are probably Windows user and `CTRL + Z`/ `Ctrl + Pause` / `Ctrl + Pause` will fix the issue)
> Argh, I’m an awful person, but lowercase ctrl with uppercase C is even more confusing – if we are to keep uppercase C, then let’s keep uppercase CTRL. And then I’ll merge, promise!
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/railsgirls/railsgirls.github.com/pull/255/files#r39851376>.
>